docu/Concepts/BusinessRequirements/RegelnDerGeldschoepfung.md

```diff
@@ -1,3 +1,22 @@
 # Regeln der Geldschöpfung
 
-*Hier werden die fachlichen Regeln der 3-fachen Schöpfung beschrieben*
+Nach der Gradido-Philosophie erfolgt pro Monat der Prozess der Dreifachen-Geldschöpfung. Das bedeutet, dass in einer Gradido-Community pro Mitglied, das eine *natürliche Person* ist, eine maximale Summe von 3.000 GDD geschöpft wird. Davon kann das Mitglied 1.000 GDD als *Aktives Grundeinkommen* durch Community nützige Leistungen auf sein persönliches AGE-Konto erhalten. 1.000 GDD werden auf das Gemeinwohlkonto der Community gutgeschrieben und weitere 1.000 GDD fließen auf das AUF-Konto der Community.
```

Comment:
Hier wird immer ausgespart wie sich die 3x 1000 GDD zueinander verhalten. Als ich Bernd gefragt habe sagte er, das aus Planungssicherheit immer die vollen 2000 GDD für die Community geschöpft werden. Das halte ich allerdings für sehr gefährlich, da es dazu verleitet einfach weitere Konten zu registrieren. Hinter dem geschöpftem Geld steht dann auch ggf keine Leistung und damit keine Deckung. Ich halte es für essentiell die Schöpfung für AUF & Community GDD auch an die konkrete Schöpfung des Nutzers zu binden, da diese leistungsgedeckt ist.

Reply:
Das ist eine berechtigte Frage und muss echt konzeptionell weiter durchdacht werden.
Ein Hinweis zu dem Thema wäre noch, dass egal ob und wieviel eine Person in dem Giralgeldsystem verdient, die Kommune, in der diese Person den 1.Wohnsitz hat, auch Steuergelder zugewiesen bekommt.